kbase-extension/static/kbase/js/util/jobLogViewer.js

```diff
@@ -968,6 +971,11 @@ define([
                     message.jobState && message.jobState.status ? message.jobState.status : null,
                 { mode } = fsm.getCurrentState().state;
 
+            if (jobStatus === 'does_not_exist') {
```

Comment:
handle job not found errors -- ready for removing the `job-does-not-exist` message type